src/posts/lucky_1.1.0.cr

```diff
@@ -0,0 +1,210 @@
+    as a few changes in Lucky v1.1.0 were around working with HTMX. For example, when using `hx-boost`,
+    you must return a 303 redirect response. One change made in this version of Lucky allows you to
+    set your default redirect status to 303 globally. Set it once and forget it.
+
+    ### Changes to LuckyTask
+
+    [LuckyTask](https://github.com/luckyframework/lucky_task) received a bit of an updated too!
+    If you're unfamiliar with this shard, it's the backbone to creating custom CLI tasks. Lucky
+    ships with several pre-built tasks like `lucky db.create`, `lucky routes`, and even `lucky dev`.
+
+    When designing your custom task, you may have wanted to use `--name` as a flag, but previously,
+    this wasn't possible. Because the CLI args created instance methods, and we had several instance
+    methods reserved, this would cause some conflicts. To fix this, it required a bit of a **breaking change**
+    to the structure.
+
+    Before the update, a custom task would look like this:
+
+    ```crystal
+    class GenerateSitemaps < LuckyTask::Task
+      summary "Generates new sitemaps"
+      name "gen.sitemaps"
+
+      def help_message
+        "Run 'lucky gen.sitemaps' to refresh the sitemaps"
+      end
+
+      def call
+        # ...
+      end
+    end
+    ```
+
+    After the update:
+
+    ```crystal
+    class GenerateSitemaps < LuckyTask::Task
+      summary "Generates new sitemaps"
+      name "gen.sitemaps"
+      help_message "Run 'lucky gen.sitemaps' to refresh the sitemaps"
+
+      def call
+        # ...
+      end
+    end
+    ```
+
+    The three methods here `summary`, `name`, and `help_message` were all instance methods.
+    By moving `help_message` to a macro, these three now generate class level methods all
+    prefixed with `task_`. (i.e. `task_summary`, `task_name`, `task_help_message`)
+
+    ### JSON Serialized Arrays
+
+    In a previous release of Avram, we added the ability to set columns as a serialized column.
+    When using the `serialize: true` option on a JSON::Any column, it will serialize that column
+    in to an object.
+
+    ```crystal
+    class User < BaseModel
+      struct Preferences
+        include JSON::Serializable
+
+        property theme : String
+      end
+
+      table do
+        column preferences : User::Preferences, serialize: true
+      end
+    end
+    ```
+
+    With Avram v1.1.0, you can now store an array of these objects.
+
+    ```crystal
+    class User < BaseModel
+      struct Contact
+        include JSON::Serializable
+
+        property name : String
+      end
+
+      table do
+        column contacts : Array(User::Contact), serialize: true
+      end
+    end
+    ```
+
+    > Note: this stores the column as `'[]'::jsonb` which is a jsonb array of objects
+    > as opposed to `[]::jsonb[]` which is an array column of jsonb objects. The column
+    > is still `jsonb`.
```

Comment:
I used to use blockquotes as a callout, but I realized it's meant for quoting. Better to use _emphasis_, or just use plaintext.

```suggestion
    _Note: this stores the column as `'[]'::jsonb` which is a jsonb array of objects
    as opposed to `[]::jsonb[]` which is an array column of jsonb objects. The column
    is still `jsonb`._
```